src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,8 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts = [] }) => (
+  <div className="PostList">
+    {posts.length > 0
+      && posts.map(post => <PostInfo post={post} key={post.id} />)}
+  </div>
```

Comment:
Well done on checking the existence of posts, but this way if there are no posts the `<div className="PostList"></div>` will still render. Let's not render the whole `PostList` component if there is no posts